group.go

```diff
@@ -53,11 +53,18 @@ type Scalar interface {
 	// Set to a fresh random or pseudo-random scalar
 	Pick(rand cipher.Stream) Scalar
 
-	// SetBytes will take bytes and create a scalar out of it
+	// SetBytes sets the scalar from a big-endian byte-slice,
+	// reducing if necessary to the appropriate modulus.
 	SetBytes([]byte) Scalar
 
-	// Bytes returns the raw internal representation
+	// Bytes returns a big-Endian representation of the scalar
 	Bytes() []byte
+
+	// Allow or disallow use of faster variable-time implementations
+	// of operations on this Point, returning the old flag value.
+	// This flag always defaults to false (constant-time only)
+	// in implementations that can provide constant-time operations.
+	SetVarTime(varTime bool) bool
```

Comment:
I see, it's a valid counter argument indeed. 

I am totally in favor of dropping variable time implementation for `Scalar`, following in spirit the well-coded scalar implementation in [dalek](https://github.com/isislovecruft/curve25519-dalek/blob/master/src/scalar.rs) for example.
From my experience with this library and the algorithms I coded, I don't think it should be *that* hard to distinguish between sensitive or non sensitive scalars if one really takes the time at each individual operations. Nevertheless, my main concern is that screw-ups *can* and *will* happen (Murphy's law!). Removing vartime implementation will at least prevent a large range of potential weaknesses in existant and future implementations: that's a **huge** benefit.